vorestation.dme

```diff
@@ -2360,6 +2360,7 @@
 #include "code\modules\mob\living\carbon\human\species\station\station_special_abilities_vr.dm"
 #include "code\modules\mob\living\carbon\human\species\station\station_special_vr.dm"
 #include "code\modules\mob\living\carbon\human\species\station\station_vr.dm"
+#include "code\modules\mob\living\carbon\human\species\station\xenochimera_hud.dm"

```

Comment:
This should probably have _vr appended to it.

Reply:
Xenochim are _vr only thing anyway, but done